utils/src/main/kotlin/Utils.kt

```diff
@@ -19,6 +19,7 @@
 
 package com.here.ort.utils
 
+import ch.frankel.slf4k.info
```

Comment:
Nit: Empty line after this one.
Commit message nit: It's "a utility".

Reply:
Also, use a wildcard import here.